stack.py

```diff
@@ -1,14 +1,35 @@
-from collections import deque
-stack = deque()
+# Stack implementation in python
 
-stack.append('7')
-stack.append('6')
-stack.append('5')
 
-print(stack)
+# Creating a stack
+def create_stack():
+    stack = []
+    return stack
 
-# deque(['a', 'b', 'c'])
 
-stack.pop()
+# Creating an empty stack
+def check_empty(stack):
+    return len(stack) == 0
 
-print(stack)
\ No newline at end of file
+
+# Adding items into the stack
+def push(stack, item):
+    stack.append(item)
+    print("pushed item: " + item)
```

Comment:
@AshabaJasper , why are we using string concatenation rather than formatting

Reply:
It will be worked on Mr.Hassan

Reply:
OKay